static/monorail-2.0.yaml

```diff
@@ -308,6 +308,7 @@ definitions:
         type: string
       username:
         type: string
+      pattern: ^([A-Za-z][0-9A-Za-z._-]+)$
```

Comment:
interesting. i tested the change on box and did not receive any complaints from on-http.
i will make the change and resubmit. thanx!